strings.js

```diff
@@ -0,0 +1,173 @@
+// # String Exercises
```

Comment:
Overall good demonstration of string methods and properties. Keep up the good work and looking forward to see what you will create with hangman.